third_party/terraform/utils/config_test.go

```diff
@@ -90,6 +90,34 @@ func TestAccConfigLoadValidate_credentials(t *testing.T) {
 	}
 }
 
+func TestAccConfigLoadValidate_impersonated(t *testing.T) {
+	if os.Getenv(TestEnvVar) == "" {
+		t.Skip(fmt.Sprintf("Network access not allowed; use %s=1 to enable", TestEnvVar))
+	}
+	testAccPreCheck(t)
+
+	serviceaccount := multiEnvSearch([]string{"GOOGLE_IMPERSONATED_SERVICE_ACCOUNT"})
```

Comment:
Should this be `GOOGLE_IMPERSONATE_SERVICE_ACCOUNT`? Or do you want to change it to `IMPERSONATED` in the provider? Also, should we maybe expose this environment variable in the docs as well?

Reply:
This is a typo. It should be without the D. I'll update the docs to mention the variables too.

Reply:
fixed